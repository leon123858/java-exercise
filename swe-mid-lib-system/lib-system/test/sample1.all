# 各種合法addBook、listAuthor
3
AuthorA SubjectA #1
AuthorB SubjectB #2
AuthorC SubjectC #3
2
Staff UserA
Borrower UserB 2
UserA listAuthor AuthorB # ID: 2 Author: AuthorB Subject: SubjectB
UserA listAuthor AuthorD # Error
UserA addBook
AuthorD SubjectD # 4
UserA listAuthor AuthorD # ID: 4 Author: AuthorD Subject: SubjectD
UserA addBook
AuthorA SubjectE # 5
UserA listAuthor AuthorA # ID: 1 Author: AuthorA Subject: SubjectE/nID: 5 Author: AuthorA Subject: SubjectE
UserA addBook
AuthorK subjectA #6
UserA listAuthor AuthorK # ID: 6 Author: AuthorK Subject: SubjectA
