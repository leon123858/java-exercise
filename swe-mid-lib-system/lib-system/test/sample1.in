3
AuthorA SubjectA
AuthorB SubjectB
AuthorC SubjectC
2
Staff UserA
Borrower UserB 2
UserA listAuthor AuthorB
UserA listAuthor AuthorD
UserA addBook
AuthorD SubjectD
UserA listAuthor AuthorD
UserA addBook
AuthorA SubjectE
UserA listAuthor AuthorA
UserA addBook
AuthorK subjectA
UserA listAuthor AuthorK