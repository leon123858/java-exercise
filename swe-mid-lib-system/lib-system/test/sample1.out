ID: 2 Author: AuthorB Subject: SubjectB
Error
ID: 4 Author: AuthorD Subject: SubjectD
ID: 1 Author: AuthorA Subject: SubjectA
ID: 5 Author: AuthorA Subject: SubjectE
ID: 6 Author: AuthorK Subject: SubjectA